### ISSUE TITLE
Memory Usage

### ISSUE THREAD

**Issue by huangjy-pku** [Qi-Zhangyang/GPT4Scene#3]
Great work! I noticed that GPT4Scene is trained with a batch size of 128 (16 per GPU). Given that previous similar works (*e.g.*, LLaVA-3D, Video-3D LLM) can only support *batch size = 2 per GPU*, have you made any special operations to spare the GPU memory usage to afford such a larger batch size?

**Comment by rookiexiong7** [comment]
Thank you for your attention in our work! For the base version of Qwen2-VL-7B (GPT4Scene), we set the `gradient accumulation steps` to `4` and the `batch size per device` to 4  to minimize memory consumption. For the HD and HDM versions, we reduced the batch size and increased the number of devices to accommodate the longer context length. 

Actually, in our simple tests, the size of the batch size did not significantly affect the performance of the model
